### PR TITLE
[cmake] partners_api & traffic

### PR DESCRIPTION
**Pull request by mapsmetest**
With the corresponding tests.

### CODE REVIEW

File: traffic/traffic_tests/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+include_directories(${OMIM_ROOT}/3party/jansson/src)
+
+set(
+  SRC
+  traffic_info_test.cpp
+)
+
+omim_add_test(${PROJECT_NAME} ${SRC})
+omim_link_libraries(
```
Comment (ygorshenin) [inline traffic/traffic_tests/CMakeLists.txt]: Add blank line, please.

**Comment by ygorshenin** [comment]
LGTM with nit.

File: partners_api/partners_api_tests/CMakeLists.txt [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  jansson
+  stats_client
+  ${LIBZ}
+  ${Qt5Core_LIBRARIES}
+)
+
+if (PLATFORM_MAC)
+  omim_link_libraries(
+    ${PROJECT_NAME}
+    ${Qt5Widgets_LIBRARIES}
```
Comment (ygorshenin) [inline partners_api/partners_api_tests/CMakeLists.txt]: As far as I know @syershov insists that we need some kind of macro (or function) omim_link_with_platform(), what will include all needed frameworks and Qt5 parts.

Comment (mapsmetest) [inline partners_api/partners_api_tests/CMakeLists.txt]: I think we'd better do it in a separate pool-request.

**Comment by syershov** [comment]
LGTM

File: partners_api/partners_api_tests/CMakeLists.txt (hunk 2 above)
Comment (ygorshenin) [inline partners_api/partners_api_tests/CMakeLists.txt]: Бассейн-запрос? :)

**Pull request merged**